src/index.html

```diff
@@ -11,7 +11,217 @@
+        </div>
+
+      </fieldset>
+      <fieldset class="fieldset registration">
+        <legend>
+          Registration
+        </legend>
+        <div class="block reg__info">
+          <label for="email">
+            E-mail:
+          </label>
+          <input 
+            id="email" 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required 
+            multiple
+            autocomplete="off"
+          >
+        </div>
+        
+        <div class="block reg__info">
+          <label for="password">
+            Password:
+          </label>
+          <input 
+            id="password"
+            type="password"
+            name="password"
+            minlength="10"
+            required
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset facts">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="block fact__about">
+      
+          <label for="YesOrNot">
+             Do you love cats?
+            <input 
+              id="YesOrNot"
+              type="radio"
+              name="YesOrNot"
+              value="yes"
+            >
+             Yes
+          </label>
+          <label>
+            <input 
+              id="YesOrNot1"
+              type="radio"
+              name="YesOrNot"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="block fact__about">
+          <label for="color">
+            What is your favorite color?
+            <input 
+              id="color" 
+              type="color"
+              name="color" 
+              required
+            >
+          </label>
+        </div>
+
+        <div class="block fact__about">
+          <label for="time">What time do you go to bed?</label>
+          <input 
+            id="time"
+            type="time"
+            name="time"
+            required
+          >
+        </div>
+
+        <div class="block fact__about">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select 
+            id="cars" 
+            name="cars" 
+            multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="block fact__about">
+          <label for="rate">How do you rate our work?</label>
+          <input 
+            id="rate" 
+            type="range" 
+            name="range"
+          >
+        </div>
+      </fieldset>
+      <fieldset 
+       class="fieldset additional__information">
+        <legend>Additional info:</legend>
+        <div class="block add_info">
+          <label for="comments">Comments:</label>
+          <textarea
+            id="comments"
+            name="comments">
+          </textarea>
```

Comment:
use this trick for textarea to avoid empty space when focus at first time
```suggestion
          <textarea
            id="comments"
            name="comments"></textarea>
```